thingsboard_gateway/connectors/ftp/ftp_connector.py

```diff
@@ -115,16 +116,20 @@ def run(self):
                             if self.__stopped:
                                 break
         except Exception as e:
-            self.__log.exception(e)
+            self.__log.error("Unexpected exception in loop for %s %d with error %s" % (self.host, self.port, str(e)))
```

Comment:
Please, use the following format of calling logs method, it will avoid unnecessary formatting:

"Unexpected exception in loop for %s %d with error %r", self.host, self.port, e)